sessionctx/variable/sysvar.go

```diff
@@ -565,7 +566,7 @@ var defaultSysVars = []*SysVar{
 	{Scope: ScopeSession, Name: WarningCount, Value: "0", ReadOnly: true},
 	{Scope: ScopeSession, Name: ErrorCount, Value: "0", ReadOnly: true},
 	{Scope: ScopeGlobal | ScopeSession, Name: WindowingUseHighPrecision, Value: BoolOn, Type: TypeBool, IsHintUpdatable: true},
-	{Scope: ScopeSession, Name: TiDBTxnScope, Value: config.GetTxnScopeFromConfig()},
+	{Scope: ScopeSession, Name: TiDBTxnScope, Value: oracle.GlobalTxnScope},
```

Comment:
Now the default value of `txn_scope` is always `global`. Users must set the session variable every time before using local transactions, is it right?

Reply:
yes

Reply:
Is it inconsistent with the previous？If dc is set, it will default to local?

Reply:
If that is a concern, we can set this the default value to `local` if `zone` is not empty, otherwise `global`.

Reply:
Got it.

Reply:
updated.